docs/README.md

```diff
@@ -1,195 +1,83 @@
-# VTEX Minicart
+📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

Comment:
I think there's a lot of places with that link being wrong. I just noticed this yesterday.

```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/minicart) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

Reply:
Woow! Good catch!